model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -105,6 +105,8 @@
         "code-llama-7b": "codellama/CodeLlama-7b-hf",
         "code-llama-13b": "codellama/CodeLlama-13b-hf",
         "code-llama-34b": "codellama/CodeLlama-34b-hf",
+        "llm-jp-13b-instruct-full-jaster-v1.0": "llm-jp/llm-jp-13b-instruct-full-jaster-v1.0",
```

Comment:
is there a better canonical name? the current names are pretty verbose

Reply:
I agree, it's pretty bad nomenclature but they want to have 2 different llm-jp-13b-instruct models for some reason

Reply:
maybe we could have the model named `llm-jp-13b-instruct-full` and `llm-jp-13b-instruct-full-dolly`?

Reply:
Yeah I think those names seem reasonable.